src/layout.js

```diff
@@ -21,7 +21,10 @@
 
 import {dev, user} from './log';
 import {isFiniteNumber} from './types';
+import {isExperimentOn} from './experiments';
 
+/** @private @const {string} */
+const UX_EXPERIMENT = 'amp-ad-loading-ux';
```

Comment:
We can't do that because: https://github.com/ampproject/amphtml/blob/master/build-system/dep-check-config.js#L166
The opposite will work though. But why do we need it to import from `layout.js` file. It's only a string.